packages/itwin/tree-widget/src/types.ts

```diff
@@ -35,7 +35,7 @@ export interface ModelTreeProps extends Omit<ModelsTreeProps,
 | "height"
 | "filterInfo"
 | "onFilterApplied"
-> { TreeHeaderButtons?: Array<(props: ModelsTreeHeaderButtonProps) => React.ReactNode> }
+> { treeHeaderButtons?: Array<(props: ModelsTreeHeaderButtonProps) => React.ReactNode> }
```

Comment:
maybe change `treeHeaderButtons` to `headerButtons`? interface name already tells that's the attribute is related to a tree

Reply:
fixed